allensdk/brain_observatory/ecephys/write_nwb/_schemas.py

```diff
@@ -25,8 +25,12 @@ class Channel(RaisingSchema):
     local_index = Int(required=True)
     probe_vertical_position = Int(required=True)
     probe_horizontal_position = Int(required=True)
-    manual_structure_id = Int(required=True, allow_none=True)
-    manual_structure_acronym = String(required=True, allow_none=True)
+    structure_id = Int(required=True, allow_none=True)
+    structure_acronym = String(required=True, allow_none=True)
+    AP_coordinate = Float(required=True, allow_none=True)
```

Comment:
This will fail without lims changes. If you'd like, we can sit down together and make those changes.